projects/DensePose/doc/TOOL_APPLY_NET.md

```diff
@@ -34,6 +34,51 @@ python apply_net.py dump configs/densepose_rcnn_R_50_FPN_s1x.yaml DensePose_ResN
 python apply_net.py dump configs/densepose_rcnn_R_50_FPN_s1x.yaml DensePose_ResNet50_FPN_s1x-e2e.pkl "image*.jpg" --output results.pkl -v
 ```
 
+If you want to load the pickle file generated by the above command:
+```
+# make sure DensePose is in your PYTHONPATH, or use the following line to add it:
+sys.path.append("/your_detectron2_path/detectron2_repo/projects/DensePose/")
+
+f = open('/your_result_path/result.pkl', 'rb')
```

Comment:
Please use "results.pkl" (also below) because that's what the above command uses.